lib/prometheus/engineering_collector.rb

```diff
@@ -1,12 +1,17 @@
 # frozen_string_literal: true
 
+require 'lru_redux'
+
 unless defined? Rails
   require File.expand_path('../../config/environment', __dir__)
 end
 
 # when /metrics is called. This runs directly on prometheus_exporter.
 class EngineeringCollector < PrometheusExporter::Server::TypeCollector
+  LRU_MINUTES = 15
```

Comment:
I think for these specific metrics below the number can be much higher. I would go with 60 minutes, but having like 12 hours is also okay from the data integrity side. On the other hand, why should we keep something in-memory for days :shrug:

Reply:
One hour it is.